dtfs-central-api/src/services/repositories/utilisation-reports-repo/utilisation-reports-repo.ts

```diff
@@ -32,15 +32,15 @@ export const saveUtilisationReportDetails = async (
   };
 
   const utilisationReportDetailsCollection = await db.getCollection(DB_COLLECTIONS.UTILISATION_REPORTS);
-  const updatedResult = await utilisationReportDetailsCollection.updateOne(
+  await utilisationReportDetailsCollection.updateOne(
     {
       _id: { $eq: reportId },
-      'reportPeriod': { $eq: reportPeriod },
+      reportPeriod: { $eq: reportPeriod },
```

Comment:
Decided to replace this with matching the whole report that we found in the function which calls this method

Reply:
This seems a bit overkill as the `_id` is already unique to the report, so this will just slow down the query.

I don't think we need to worry about the case where the report has been updated since we originally fetched it, but if for some reason it didn't match the full report I think it would just fail silently anyway? If we did want to think about this case we'd probably take a different approach, like locking the row while we update.

Reply:
A question for when we refactor to SQL I think 🙃 - I did replace the filter with the original "id only" check though